### DISCUSSION TITLE
Is V2 version already stopped maintaining?

### DISCUSSION THREAD

**Discussion by rofixro** [realworld-apps/realworld#1376] [General]
As titled

**Comment by geromegrignon** [comment]
Hi, not at all, but it takes time.
Latest task was to create a new version of the frontend part of the demo (https://github.com/gothinkster/realworld/tree/main/apps/demo) as the original one was created with AngularJS (https://github.com/gothinkster/angularjs-realworld-example-app).

It'll be easier to manage the following tasks by having the whole core of the project in the same repository by now.

**Reply by rofixro** [reply]
Where can I see the progress on the frontend part? For example, future plans or unfinished tasks?

**Reply by geromegrignon** [reply]
There is this changelog: https://github.com/gothinkster/realworld/issues/808
But I haven't updated it for a while, and having the demo front and back projects in isolated repositories was not easy to manage the progress. As it's done, i can focus on the big picture.

I'll focus on updating it during the weekend and creating a dedicated GitHub board project.
